src/gtfs/read.rs

```diff
@@ -744,10 +745,11 @@ fn get_modes_from_gtfs(
         .iter()
         .map(|mt| get_commercial_mode(mt))
         .collect();
-    let physical_modes = gtfs_mode_types
+    let physical_modes_set: HashSet<objects::PhysicalMode> = gtfs_mode_types
         .iter()
         .map(|mt| get_physical_mode(mt))
         .collect();
+    let physical_modes = physical_modes_set.into_iter().collect();
```

Comment:
```rust
let physical_modes = gtfs_mode_types
    .iter()
    .map(|mt| get_physical_mode(mt))
    .collect::<BTreeSet<_>>()
    .into_iter()
    .collect();
```

BTreeSet because it will have a deterministic order.

Reply:
order by id? or co2_emission?

Reply:
order by `Ord`